ci-operator/step-registry/telco5g/sno-ztp-cluster-setup/telco5g-sno-ztp-cluster-setup-commands.sh

```diff
@@ -0,0 +1,324 @@
+    shell: cp /home/kni/ztp-jobs/${SNO_NAME}/config/auth/kubeconfig /home/kni/.kube/config_${SNO_NAME}
+
+  - name: Copy kubeconfig from ZTP directory
+    shell: cp /home/kni/ztp-jobs/${SNO_NAME}/ztphub/out/${CLUSTER_NAME}-kubeadmin-kubeconfig /home/kni/.kube/ztp_config_${CLUSTER_NAME}
+
+  - name: Add skip-tls-verify to mgmt kubeconfig
+    replace:
+      path: /home/kni/.kube/config_${SNO_NAME}
+      regexp: '    certificate-authority-data:.*'
+      replace: '    insecure-skip-tls-verify: true'
+
+  - name: Add skip-tls-verify to ZTP kubeconfig
+    replace:
+      path: /home/kni/.kube/ztp_config_${CLUSTER_NAME}
+      regexp: '    certificate-authority-data:.*'
+      replace: '    insecure-skip-tls-verify: true'
+
+  - name: Grab the kubeconfig of management cluster
+    fetch:
+      src: /home/kni/.kube/config_${SNO_NAME}
+      dest: $SHARED_DIR/mgmt-kubeconfig
+      flat: yes
+
+  - name: Grab the ZTP kubeconfig
+    fetch:
+      src: /home/kni/.kube/ztp_config_${CLUSTER_NAME}
+      dest: $SHARED_DIR/kubeconfig
+      flat: yes
+
+  - name: Modify local copy of mgmt kubeconfig
+    replace:
+      path: $SHARED_DIR/mgmt-kubeconfig
+      regexp: '    server: https://api.*'
+      replace: "    server: https://${SNO_IP}:${SNO_CLUSTER_API_PORT}"
+    delegate_to: localhost
+
+  - name: Modify local copy of ZTP kubeconfig
+    replace:
+      path: $SHARED_DIR/kubeconfig
+      regexp: '    server: https://(.*):6443'
+      replace: '    server: https://${CLUSTER_API_IP}:${CLUSTER_API_PORT}'
+    delegate_to: localhost
+
+  - name: Add docker auth to enable pulling containers from CI registry
+    shell: >-
+      oc --kubeconfig=/home/kni/.kube/config_${SNO_NAME} set data secret/pull-secret -n openshift-config --from-file=.dockerconfigjson=/home/kni/pull-secret.txt
+
+  - name: Add docker auth to enable pulling containers from CI registry for ZTP cluster
+    shell: >-
+      oc --kubeconfig=/home/kni/.kube/ztp_config_${CLUSTER_NAME} set data secret/pull-secret -n openshift-config --from-file=.dockerconfigjson=/home/kni/pull-secret.txt
+
+  - name: Patching hostedcluster to disable all default sources
+    shell: >-
+      oc --kubeconfig=/home/kni/.kube/config_${SNO_NAME} patch hostedcluster ${CLUSTER_NAME} -n clusters --type=merge -p '{"spec": {"configuration": {"operatorhub": {"disableAllDefaultSources": true}}}}'
+
+EOF
+
+# Run the playbook to install the cluster
+echo "Run the playbook to install the cluster"
+status=0
+
+ANSIBLE_LOG_PATH=$ARTIFACT_DIR/ansible.log ANSIBLE_STDOUT_CALLBACK=debug ansible-playbook \
+    playbooks/sno_ztp.yml \
+    -e ztphost=$CLUSTER_NAME \
+    -e vsno_name=$SNO_NAME \
+    -e vsno_ip=$SNO_IP \
+    -e hostedbm_inject_dns=true \
+    -e sno_tag=$MGMT_VERSION \
+    -e vsno_wait_minutes=150 \
+    -e sno_release=$T5CI_JOB_MGMT_RELEASE_TYPE \
+    -e ztp_tag=$T5CI_VERSION \
+    -e ztp_release=$T5CI_JOB_ZTP_RELEASE_TYPE $PLAYBOOK_ARGS || status=$?
```

Comment:
seems like 60 mins is not enough, let's add more time to wait: `-e ztphub_wait_install_timeout=90`

Reply:
@danimjaffe You need to set two vars for disconnected:-

enable bool to use disconnected and path to pull secret file on hypervisor

-e disconnected_env=true 
-e disconnected_env_pull_secret=<>/pull-secret.txt